extensions/shared/paid-blocks/upgrade-plan-banner.jsx

```diff
@@ -41,15 +41,15 @@ const UpgradePlanBanner = ( {
 	 * It should be updated if https://github.com/WordPress/gutenberg/pull/23993 is merged.
 	 */
 	useEffect( () => {
-	    if ( ! bannerRef?.current ) {
-	    	return;
-	    }
-	    const inspectorEl = bannerRef.current.closest( '.block-editor-block-inspector' );
-	    if ( ! inspectorEl ) {
-	    	return;
-	    }
-	    inspectorEl.classList.add( 'with-upgrade-plan-banner' );
-	    inspectorEl.classList.add( `is-${ blockName }-premium-block` );
+		if ( ! bannerRef?.current ) {
+			return;
+		}
+		const inspectorEl = bannerRef.current.closest( '.block-editor-block-inspector' );
+		if ( ! inspectorEl ) {
+			return;
+		}
+		inspectorEl.classList = [ 'block-editor-block-inspector' ];
```

Comment:
why do you set the classes here?

Reply:
It seems that the `( '.block-editor-block-inspector' )` element is not removed from DOM resulting in having various classes from all different paid blocks. 
`inspectorEl.classList = [ 'block-editor-block-inspector' ];` acts like a reset, by setting the initial class before adding the new. It is a little hacky, but I assume the whole implementation of changing the banner order in the sidebar is hacky and we should not investigate more time on it?

Reply:
Fixed in https://github.com/Automattic/jetpack/pull/16495/commits/49d97f304e51f805ba151be5b118c5e578022937